moabb/paradigms/p300.py

```diff
@@ -151,9 +151,7 @@ def process_raw(  # noqa: C901
             events = mne.find_events(raw, shortest_event=0, verbose=False)
         else:
             try:
-                events, _ = mne.events_from_annotations(
-                    raw, event_id=event_id, verbose=False
-                )
+                events, _ = mne.events_from_annotations(raw, event_id=None, verbose=False)
```

Comment:
Not sure this is a good idea. With `event_id=None`, any description will be map an integer-based code, based on their sorted number. It will affect all P300 datasets and change their event_id values.

Reply:
I see. For Sosulski2019 dataset annotation descriptions are: {'New Segment/': 1, 'Stimulus/S  1': 2, 'Stimulus/S 21': 3, 'Stimulus/S255': 4} and for Huebner2017 and Huebner2018 {'Stimulus/S   0': 1, 'Stimulus/S   1': 2}. 
Then, event_id = self.used_events(dataset) corresponds to {'Target': dataset.event_id['Target'], 'NonTarget': dataset.event_id['NonTarget'] }, so with events, _ = mne.events_from_annotations(raw, event_id=event_id, verbose=False) there is no matching annotations.

Reply:
Since for all three datasets the files are Brain Vision, changing events, _ = mne.events_from_annotations(raw, event_id=**event_id**, verbose=False) **to** events, _ = mne.events_from_annotations(raw, event_id=**'auto'**, verbose=False) and preserving initial event ids (10002 and 10001; and 21 and 1) might be a better solution?

Reply:
yes, this is correct